deb_files/download_snapshot.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+if [ ! -f latest-export.gz.bin ]; then
+	wget https://dbfiles.iota.org/mainnet/hornet/latest-export.gz.bin
+fi
```

Comment:
I was thinking about a "run.sh" script, which asks the user if he wants to delete the mainnetdb. If so, it checks automatically if the latest snapshot is still the latest. I have some codesnippets for you if you want to do that? :)

`run.sh`
```sh
#!/bin/bash
service hornet stop

echo "Do you want to delete the old database?"
select yn1 in "Yes" "No"; do
    case $yn1 in
        Yes ) rm -Rf mainnetdb/; ./download_snapshot.sh; break;;
         No ) break;;
    esac
done

service hornet start
journalctl -u hornet -f
```

`download_snapshot.sh`
```sh
#!/bin/bash

finalurl() { curl --silent --location --head --output /dev/null --write-out '%{url_effective}' -- "$@"; }
checkIfModified() { curl --silent --output /dev/null --write-out '%{http_code}' -I --header 'If-Modified-Since: Tue, 17 Dec 2019 11:56:38 GMT' -- "$@"; }

lastest_file=`finalurl https://dbfiles.iota.org/mainnet/hornet/latest-export.gz.bin`
status_code=`checkIfModified ${lastest_file}`
if [[ "$status_code" -ne 304 ]] ; then
    echo "Snapshot outdated, downloading snapshot..."
    rm latest-export.gz.bin
    wget ${lastest_file}
else
    echo "Snapshot up to date"
fi
```